SVG.flood/svg.svg.flt

```diff
@@ -11,8 +11,10 @@
     <% insert("definitions/fillImageDefinitions.svg.flt", indentation: 1) %>
     <!-- Gradients -->
     <% insert("definitions/gradientDefinitions.svg.flt", indentation: 1) %>
+    <% if isShadowVisible(layer: layer, timeline: timeline) %>
     <!-- Shadows -->
-    <% insert("definitions/shadowDefinitions.svg.flt", indentation: 1) %>
+    <% insert("definitions/shadowDefinitions.svg.flt", indentation: 1)
+       end %>
```

Comment:
move this inside  `definitions/shadowDefinitions.svg.flt` and include shadows on a layer per layer basis